### PR TITLE
Added dense qrcodes functionality, with inverter 'i' code

### PR DESCRIPTION
**Pull request by queenp**
So far as I can tell this works.

### CODE REVIEW

**Comment by queenp** [comment]
fix for #7962

**Comment by XVilka** [comment]
I was thinking, what if just reuse `scr.utf8` config value instead of inventing a new one?

**Comment by XVilka** [comment]
I mean to use this dense QR code generation by default, if `scr.utf8` is enabled.

**Comment by queenp** [comment]
Hi @XVilka, I'd be perfectly happy with that suggestion, I think it makes most sense also (I can't see a reason why you'd have large UTF8-block characters in your font and not the half-height ones used above for the half-pitch UTF-8 rendering), I was just following the issue spec (more or less) because I'm a new contrib and still finding my way around the project/codebase and I didn't want to assume I knew better just because.

Also, I don't know whether people are happy with my (admittedly almost certainly useless) half-pitch ascii pixels, if it should be scrapped or if it shoudl be default dense for utf8 and default-large for ASCII.

So, essentially I'm happy to trust you folks judgment on this and tweak it as necessary.

**Comment by radare** [comment]
yep , im all with @XVilka honoring the utf8 var

File: libr/util/qrcode.c [hunk 1]
```diff
@@ -1026,16 +1036,39 @@ R_API char *r_qrcode_gen(const ut8 *text, int len, bool utf8) {
...
 	int x, y;
-	for (y = -border; y < size + border; y++) {
-		for (x = -border; x < size + border; x++) {
-			bool fill = qrcodegen_getModule (qrcode, x, y);
-			const char *pixel = utf8
-				? (fill? "██": "  ")
-				: (fill? "##": "  ");
-			memcpy (p, pixel, strlen (pixel));
-			p += strlen (pixel);
+	if(denseqr) {
```
Comment (radare) [inline libr/util/qrcode.c]: missing spaces before every ( and around += and such

File: libr/util/qrcode.c [hunk 2]
```diff
@@ -1026,16 +1036,39 @@ R_API char *r_qrcode_gen(const ut8 *text, int len, bool utf8) {
...
+				const char *pixel = utf8
+					? qrcode_utf8_expansions[
+						inverted
+						? 15 - bmp
+						: bmp]
+					: qrcode_ascii_expansions[
+						inverted
+						? 15 - bmp
+						: bmp];
+				memcpy(p,pixel,strlen(pixel));
```
Comment (radare) [inline libr/util/qrcode.c]: missing spaces before (

**Pull request merged**